forge/src/main/java/org/openjdk/skara/forge/Review.java

```diff
@@ -57,8 +57,12 @@ public Verdict verdict() {
         return verdict;
     }
 
-    public Hash hash() {
-        return hash;
+    /**
+     * The hash for the commit for which this review was created. Can be null if the commit
+     * no longer exists.
```

Comment:
I think technically the Optional is empty, rather than "null".

Reply:
Right, in my mind I was documenting the state of the field rather than the return value of the method. I can change it.